### PR TITLE
Build: Rename the primary branch `master` to `main`

### PR DESCRIPTION
**Pull request by mgol**

### CODE REVIEW

**Review by melloware — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,10 +53,10 @@ cd jquery-ui
 git remote add upstream git://github.com/jquery/jquery-ui.git
 ```
 
-* Get in the habit of pulling in the "upstream" master to stay up to date as jQuery UI receives new commits.
+* Get in the habit of pulling in the "upstream" main to stay up to date as jQuery UI receives new commits.
```
Comment (fnagel) [inline CONTRIBUTING.md:56]: What about the addition "branch" we talked about? See https://jqueryui-dev.slack.com/archives/GPFN73HHP/p1612705325000200?thread_ts=1612544137.001000&cid=GPFN73HHP

**Review by fnagel — APPROVED** [review]
Looks good in general but I would prefer if we add "main branch" in the readme.

File: CONTRIBUTING.md (hunk 1 above)
Comment (mgol) [reply CONTRIBUTING.md:56]: Ha! I added that word to the commit message but not here. I'll do that.

Comment (mgol) [reply CONTRIBUTING.md]: @fnagel PR updated, please have a look.

Comment (fnagel) [reply CONTRIBUTING.md]: @mgol Perfect!

**Pull request merged**